tests/dummy/app/templates/demos/index.hbs

```diff
@@ -94,34 +63,9 @@
 
 <hr>
 
-<h3>Helpers</h3>
-<div class="list-group">
-    <div class="list-group-item">
-        <h6>get-key</h6>
-        <p><strong>arguments:</strong> an object, a key, a default key</p>
-        <p>The key is resolved on the object, to be returned.  If it is falsy, and a default key is supplied, the default
-        key is resolved on the object and that result is returned.</p>
-    </div>
-
-    <div class="list-group-item">
-        <h6>render-component</h6>
-        <p><strong>arguments:</strong> component name, optional bound properties</p>
-        <p>Render the component referenced by name. Bound properties can be passed to the component in the normal
-        fashion.</p>
-    </div>
-</div>
-
-<hr>
-
 <h3>Utility Classes</h3>
 <div class="list-group">
     <div class="list-group-item">
-        <h6>sl-grid-key-adapter</h6>
-        <p>Provides an abstraction between the events the <em>sl-grid system</em> component listens for and the ability to associate
-        any keyboard shortcuts in your application to trigger them.</p>
-    </div>
-
-    <div class="list-group-item">
         <h6>sl-menu-key-adapter</h6>
```

Comment:
This file should also be modified so that the sl-pagination-controller doesn't reference sl-grid-system